app/views/idv/doc_auth/verify.html.erb

```diff
@@ -1,5 +1,24 @@
 <div id="form-steps-wait-alert"></div>
 
+<%=  flow_session %>
+<%# if flow_session[:had_barcode_read_failure] %>
```

Comment:
this flow_session is not developed yet. When the session is updated to have :had_barcode_read_failure remove the comments for this if condition.

Reply:
Rebased and updated to use real condition in 7704bec.